homeassistant/components/wirelesstag/sensor.py

```diff
@@ -90,7 +97,11 @@ def native_value(self):
     @property
     def device_class(self):
         """Return the class of the sensor."""
-        return self._sensor_type
+        return (
+            SENSOR_TEMPERATURE
```

Comment:
This either needs to return one of the device class constants imported from the sensor integration or be validated with our sensor device class schema also imported from the sensor integration.

I recommend defining `entity_description` for each sensor type to solve this.

Example:
https://github.com/home-assistant/core/blob/4555f52e50ea543b4274473d2804e4d4219d142f/homeassistant/components/airly/sensor.py#L72

Reply:
@MartinHjelmare made changes per your suggestion. please review )